store/sqlstore/utils.go

```diff
@@ -147,3 +147,18 @@ func constructArrayArgs(ids []string) (string, []interface{}) {
 
 	return "(" + placeholder.String() + ")", values
 }
+
+// morphWriter is a target to pass to the logger instance of morph.
+// For now, everything is just logged at a debug level. If we need to log
+// errors/warnings from the library also, that needs to be seen later.
+type morphWriter struct {
+}
+
+func newMorphWriter() *morphWriter {
+	return &morphWriter{}
+}
+
+func (l *morphWriter) Write(in []byte) (int, error) {
```

Comment:
I'd suggest removing the escape sequences for colors, right now the log line looks like: 

`"morph.go:174: \u001b[36m==  upgrade_users_v6.3: migrated (0.0025s)  ========================================================\u001b[0m\n","caller":"sqlstore/utils.go:158"`

Is it something doable?

Reply:
https://github.com/go-morph/morph/pull/37 :)

Reply:
Ehe, didn't see that. Well, I think we can handle this on the morph side. Eg. if another logger passed to replace default one we can disable coloring. In general i think coloring helps.

Reply:
Yes exactly.